test/check-storage-mountpoints-btrfs

```diff
@@ -159,11 +159,9 @@ class TestStorageMountPointsBtrfs (VirtInstallMachineCase):
         disk = "/dev/vda"
         tmp_mount = "/tmp/btrfs-mount-test"
         s.partition_disk(disk, [("1MiB", "biosboot"), ("1GB", "ext4"), ("", "btrfs")])
+        s.create_luks_partition(f"{disk}3", "einszweidrei", "encrypted-vol", "btrfs")
         m.execute(f"""
-        # encrypt the btrfs partition
-        echo einszweidrei | cryptsetup luksFormat {disk}3
         echo einszweidrei | cryptsetup luksOpen {disk}3 encrypted-vol
```

Comment:
Is this line left intentionally? Shouldn't it be removed?

Reply:
Actually the `create_luks_device` helper by default closes the device  after creation, we need to have it open for manipulating it. 
however I expose `close_luks` parameter, I will utilize here :) thanks for spotting.